modules/db/src/test/java/org/mule/module/db/integration/template/TemplateQueryConfigTestCase.java

```diff
@@ -132,23 +132,6 @@ public void usesDynamicQueryWithPlaceholder() throws Exception
     }
 
     @Test
-    public void usesNullDefaultParamValue() throws Exception
```

Comment:
I removed this case because It was testing the behavior that I'm fixing.
In the config, text query references a param that isn't defined through a db-in-param element.
It must trigger an exception.
Currently, It is setting the param value as null.
If It's important setting a parameter with a null value, please let me know and I'll add this feature.